lib/auth/v4/constructStringToSign.ts

```diff
@@ -40,14 +40,6 @@ export default function constructStringToSign(params: {
         service: params.awsService,
     });
 
-    // TODO Why that line?
```

Comment:
why indeed?
and why can we now remove it?

Reply:
I removed it because after reviewing the function and checking all the types, the function cannot return an error, so this code cannot be reached.